docs/content/install/install-on-kubernetes.md

```diff
@@ -94,12 +94,6 @@ $ helm install gomods/athens-proxy -n athens --namespace athens
 
 This will deploy a single Athens instance in the `default` namespace with `disk` storage enabled. Additionally, a `ClusterIP` service will be created.
 
-By default, the chart will install Athens with a replica count of 1. To change this, change the `replicaCount` value:
-
-```console
-helm install gomods/athens-proxy -n athens --namespace athens --set replicaCount=3
-```
-
```

Comment:
@scottilee I should know this by now, but is this command somewhere else?

Reply:
If you look at the screenshot I posted they're right above and below each other. Repasting the screenshot:
<img width="1062" alt="Screen Shot 2019-04-29 at 6 16 24 PM" src="https://user-images.githubusercontent.com/7443383/56936211-f8a41c00-6aaa-11e9-9b95-59f5668c38d2.png">

I figured it was an "advanced" command so I left it in that section instead of the previous one.